rest-service/manager_rest/cloudify_acl_handler.py

```diff
@@ -0,0 +1,27 @@
+#########
+# Copyright (c) 2015 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  * See the License for the specific language governing permissions and
+#  * limitations under the License.
+
+from flask_securest.acl_handlers.abstract_acl_handler import AbstractACLHandler
+
+SECURITY_CTX_USER = 'user'
+
+
+class CloudifyACLHandler(AbstractACLHandler):
```

Comment:
I assume this is the "Default" or "Simple" ACL handler so I think it should be called as such. No need to prefix with "Cloudify" (also the module name)